playbooks/common/openshift-checks/health.yml

```diff
@@ -1,5 +1,7 @@
 ---
-- include: ../openshift-cluster/evaluate_groups.yml
+- include: ../openshift-cluster/std_include.yml
```

Comment:
Tried this locally, and I'm getting:

```
Failure summary:


  1. Hosts:    node1
     Play:     Gather and set facts for node hosts
     Task:     Warn user about bad openshift_hostname values
     Message:  The conditional check 'lookupip.stdout not in ansible_all_ipv4_addresses' failed. The error was: error while evaluating conditional (lookupip.stdout not in ansible_all_ipv4_addresses): Unable to look up a name or access an attribute in template string ({% if lookupip.stdout not in ansible_all_ipv4_addresses %} True {% else %} False {% endif %}).
               Make sure your variable name does not contain invalid characters like '-': argument of type 'StrictUndefined' is not iterable
               
               The error appears to have been in '/home/vagrant/src/github.com/openshift/openshift-ansible/playbooks/common/openshift-cluster/validate_hostnames.yml': line 12, column 5, but may
               be elsewhere in the file depending on the exact syntax problem.
               
               The offending line appears to be:
               
                   failed_when: false
                 - name: Warn user about bad openshift_hostname values
                   ^ here
```

Strangely, only `node1`  failed; other 3 hosts were okay.